tsdb/head.go

```diff
@@ -343,6 +347,14 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_mmap_chunk_corruptions_total",
 			Help: "Total number of memory-mapped chunk corruptions.",
 		}),
+		sparseHistogramsAddedTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "prometheus_tsdb_sparse_histogram_added_total",
```

Comment:
I think the `_total` suffix (and the counter type) implies that these are the number of sparse histograms addad. So I would just leave out the `added` in both the metric name and the variable name.

And (as in the variable name), plural ("histograms") reads better, I think.